x-pack/plugins/ml/server/lib/alerts/register_anomaly_detection_alert_type.ts

```diff
@@ -63,7 +147,9 @@ export function registerAnomalyDetectionAlertType({
     RuleTypeState,
     AlertInstanceState,
     AnomalyDetectionAlertContext,
-    AnomalyScoreMatchGroupId
+    AnomalyScoreMatchGroupId,
+    'recovered',
```

Comment:
should be able to use `RecoveredActionGroupId` from `'@kbn/alerting-plugin/common'`